### PR TITLE
[4.0][com_content] Remove column-related code.

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue # .

### Summary of Changes

This removes remaining code related to column options in `com_content`. See #18319 and #22840 for details.

### Testing Instructions

Code review.

### CODE REVIEW

**Comment by infograf768** [comment]
### NOTE
To @wilsonge @laoneo @bakual @mbabker @rdeutz 
Concerning `installation/src/Model/LanguagesModel.php`, what do we do with the methods which have allowed to install a basic multilingual site at installation time in 3.x as it looks like it has been apparently officially decided to drop this functionality in 4.0 in favor of the multilang sample data plugin.

I still question this decision btw. And it looks like I am not alone on this.

**Comment by Bakual** [comment]
Can you ask this question in a separate issue? It's quite offtopic here as this PR is completely unrelated to that question.

**Comment by infograf768** [comment]
I asked it here as the file has been modified and the methods may have to be totally deleted if the decision is confirmed. Just a Note to admins.

**Comment by Bakual** [comment]
Still needs an own issue to be properly discussed so it doesn't derail this valid PR here.

**Comment by infograf768** [comment]
sure

File: components/com_content/tmpl/categories/default.xml [hunk 1]
```diff
@@ -217,30 +217,13 @@
...
 				useglobal="true"
 			/>
 
-			<field 
-				name="multi_column_order" 
-				type="list"
-				label="JGLOBAL_MULTI_COLUMN_ORDER_LABEL"
-				useglobal="true"
-				>
-				<option value="0">JGLOBAL_DOWN</option>
```
Comment (Quy) [inline components/com_content/tmpl/categories/default.xml:240]: Remove these two strings?

Comment (SharkyKZ) [reply components/com_content/tmpl/categories/default.xml:240]: Removed.

**Comment by brianteeman** [comment]
Personally I would  not remove the language strings from the global language file. When I removed all unused strings earlier I left the ones in global files as those are the files recommended for extensions to utilise and I felt it was an unnecessary additional change they would have to make. My 2c

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 62d7cb19804dbe2a099da2e52bec5243423898ee<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23123">issues.joomla.org/tracker/joomla-cms/23123</a>.</sub>

**Comment by wilsonge** [comment]
@zero-24 @SniperSister RIPS please

**Comment by zero-24** [comment]
Fixed rips sorry that it took that long @wilsonge

File: components/com_content/tmpl/categories/default.xml (hunk 1 above)
Comment (infograf768) [reply components/com_content/tmpl/categories/default.xml:240]: Not sure we should delete these strings as they may be used by 3pd.

**Comment by wilsonge** [comment]
@zero-24 doesn't seem to have worked? Or there's a new issue :( No worries about the time

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**